dai_cadcad/flip.py

```diff
@@ -0,0 +1,108 @@
+""" Flip Module
+    Class-based representation of the Flip smart contract
+    (contains only what is necessary for the simulation)
+"""
+
+
+class Flip:
+    ADDRESS = "flip"
+
+    vat = None
+    beg = None
+    ttl = None
+    tau = None
+    kicks = None
+    bids = {}
+
+    def __init__(self, vat, beg, ttl, tau, kicks, bids):
```

Comment:
We're missing an `ilk` class attribute, we should include it as a parameter to the `__init__` function & instantiate it there